### ISSUE TITLE
Crash on opening save from new update [v0.9.0]

### ISSUE THREAD

**Issue by DarkCeptor44** [Goz3rr/SatisfactorySaveEditor#151]
The editor crashed when I tried opening a recently-created save from Update 3 of the Experimental branch, these are the save and log:
[savefile](https://github.com/Goz3rr/SatisfactorySaveEditor/files/4188241/save.zip)

```
2020-02-11 17:42:39.1985 - ERROR: System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. ---> System.FormatException: Input string was not in a correct format.
   at System.Version.VersionResult.SetFailure(ParseFailureKind failure, String argument)
   at System.Version.TryParseComponent(String component, String componentName, VersionResult& result, Int32& parsedComponent)
   at System.Version.TryParseVersion(String version, VersionResult& result)
   at System.Version.Parse(String input)
   at System.Version..ctor(String version)
   at SatisfactorySaveEditor.Util.UpdateChecker.VersionInfo.IsNewer()
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<CheckForUpdate>d__84.MoveNext()
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.FormatException: Input string was not in a correct format.
   at System.Version.VersionResult.SetFailure(ParseFailureKind failure, String argument)
   at System.Version.TryParseComponent(String component, String componentName, VersionResult& result, Int32& parsedComponent)
   at System.Version.TryParseVersion(String version, VersionResult& result)
   at System.Version.Parse(String input)
   at System.Version..ctor(String version)
   at SatisfactorySaveEditor.Util.UpdateChecker.VersionInfo.IsNewer()
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<CheckForUpdate>d__84.MoveNext()<---

EXCEPTION: System.FormatException: Input string was not in a correct format.
   at System.Version.VersionResult.SetFailure(ParseFailureKind failure, String argument)
   at System.Version.TryParseComponent(String component, String componentName, VersionResult& result, Int32& parsedComponent)
   at System.Version.TryParseVersion(String version, VersionResult& result)
   at System.Version.Parse(String input)
   at System.Version..ctor(String version)
   at SatisfactorySaveEditor.Util.UpdateChecker.VersionInfo.IsNewer()
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<CheckForUpdate>d__84.MoveNext()
2020-02-11 17:43:01.6464 - INFO: Opening save file: C:\Users\MSI GX63VR\AppData\Local\FactoryGame\Saved\SaveGames\c901d091044645e681969034f7d3f40f\experimental.sav
2020-02-11 17:43:01.6554 - DEBUG: Read save header: Version=LookAtTheComment, Build=22, Magic=114192, MapName=Persistent_Level, MapOpts=?startloc=Grass Fields?sessionName=experimental?Visibility=SV_FriendsOnly, Session=experimental, PlayTime=1293, SaveTime=637170503646690000
2020-02-11 17:43:01.6554 - ERROR: SatisfactorySaveParser.Exceptions.UnknownBuildVersionException: Exception of type 'SatisfactorySaveParser.Exceptions.UnknownBuildVersionException' was thrown.
   at SatisfactorySaveParser.Save.FSaveHeader.Parse(BinaryReader reader)
   at SatisfactorySaveParser.SatisfactorySave..ctor(String file)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.LoadFileAsync(String path)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<>c__DisplayClass106_0.<LoadFile>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<LoadFile>d__106.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<Open>d__101.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<<-ctor>b__82_1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
EXCEPTION: SatisfactorySaveParser.Exceptions.UnknownBuildVersionException: Exception of type 'SatisfactorySaveParser.Exceptions.UnknownBuildVersionException' was thrown.
   at SatisfactorySaveParser.Save.FSaveHeader.Parse(BinaryReader reader)
   at SatisfactorySaveParser.SatisfactorySave..ctor(String file)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.LoadFileAsync(String path)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<>c__DisplayClass106_0.<LoadFile>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<LoadFile>d__106.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<Open>d__101.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<<-ctor>b__82_1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2020-02-11 17:43:01.7152 - ERROR: SatisfactorySaveParser.Exceptions.UnknownBuildVersionException: Exception of type 'SatisfactorySaveParser.Exceptions.UnknownBuildVersionException' was thrown.
   at SatisfactorySaveParser.Save.FSaveHeader.Parse(BinaryReader reader)
   at SatisfactorySaveParser.SatisfactorySave..ctor(String file)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.LoadFileAsync(String path)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<>c__DisplayClass106_0.<LoadFile>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<LoadFile>d__106.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<Open>d__101.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<<-ctor>b__82_1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at SatisfactorySaveEditor.App.Main()
EXCEPTION: SatisfactorySaveParser.Exceptions.UnknownBuildVersionException: Exception of type 'SatisfactorySaveParser.Exceptions.UnknownBuildVersionException' was thrown.
   at SatisfactorySaveParser.Save.FSaveHeader.Parse(BinaryReader reader)
   at SatisfactorySaveParser.SatisfactorySave..ctor(String file)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.LoadFileAsync(String path)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<>c__DisplayClass106_0.<LoadFile>b__0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<LoadFile>d__106.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<Open>d__101.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at SatisfactorySaveEditor.ViewModel.MainViewModel.<<-ctor>b__82_1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at SatisfactorySaveEditor.App.Main()
```

**Comment by philomelus** [comment]
Same here.  I can provide save files if desired.

**Comment by Goz3rr** [comment]
This specific crash should be resolved in 0.9.1
If you have any other crash please make an issue and attach the save file
